restapi/src/main/java/io/stargate/web/models/udt/UdtType.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.web.models.udt;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import io.swagger.annotations.ApiModelProperty;
+import javax.validation.constraints.NotNull;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UdtType {
+  private String name;
+  private @NotNull CQLType basic;
+  private UdtInfo info;
+
+  @ApiModelProperty(value = "return the field name")
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  @JsonProperty("basic")
+  @ApiModelProperty(value = "return the field type.")
```

Comment:
```suggestion
  @ApiModelProperty(value = "The basic CQL type to be used for the field. If this type is a collection or User Defined Type then an info will also need to be provided for its type. An example of this would be a list of ints.")
```